docs/feature-flags.md

```diff
@@ -130,9 +130,6 @@ This lists the feature flags and their sub-configurations to enable/disable and
         - `dataplane-cluster-config-file` [Required]: The path to the file that contains a list of data plane clusters and their details for the service to manage (default: `'config/dataplane-cluster-configuration.yaml'`, example: [dataplane-cluster-configuration.yaml](../config/dataplane-cluster-configuration.yaml)).
     - If this is set to `auto`, the following configurations can be specified:
         - `providers-config-file` [Required]: The path to the file containing a list of supported cloud providers that the service can provision dataplane clusters to (default: `'config/provider-configuration.yaml'`, example: [provider-configuration.yaml](../config/provider-configuration.yaml)).
-        - `aws-cluster-compute-machine-type` [Optional]: The instance type of the AWS compute instances for Data Planes created in AWS (default: `m5.2xlarge`).
```

Comment:
Yes. This will be done once the SOP (see in https://issues.redhat.com/browse/MGDSTRM-6548) is done which is when we'll be in shape to turn dynamic scaling on. For now, the referenced OpenShift parameter in AppInterface will be ignored.